### PR TITLE
HotFix Button to Delete a Role 

### PR DESCRIPTION
**Pull request by fromgonie**
HotFix for the button that deletes a role

### CODE REVIEW

**Review by vascolusitan — APPROVED** [review]

File: frontend/src/router/index.ts [hunk 1]
```diff
@@ -11,6 +12,10 @@ const router = createRouter({
       path: '/',
       redirect: '/home'
     },
+    {
```
Comment (vitorlofonseca) [inline frontend/src/router/index.ts:15]: A view and route to create what?

File: frontend/src/views/HomeView.vue [hunk 2]
```diff
@@ -7,18 +7,41 @@ import { ref, onMounted } from 'vue'
 const dialogFormVisible = ref(false)
 const rolesStore = useRolesStore()
 const selectedRoleToUpdate = ref()
+const selectedRoleToDelete = ref()
+const deleteDialogForm = ref(false)
 
 const openDialog = (row: Role) => {
```
Comment (vitorlofonseca) [inline frontend/src/views/HomeView.vue:13]: Can you rename this function? Like openUpdateDialog

File: frontend/src/router/index.ts (hunk 1 above)
Comment (fromgonie) [reply frontend/src/router/index.ts:15]: This route exists for me to create roles to be easier to test the code and also for the code acceptance done by claudio.

File: frontend/src/router/index.ts [hunk 3]
```diff
@@ -12,12 +12,12 @@ const router = createRouter({
       redirect: '/home'
     },
     {
-      path: '/home',
-      component: HomeView
+      path: '/createrole',
+      component: CreateView
```
Comment (vitorlofonseca) [inline frontend/src/router/index.ts:16]: The name of the component is still unclear. CreateView means what? Create what?

Besides, the pattern of path should be /create-role

**Review by vitorlofonseca — APPROVED** [review]

**Pull request merged**